redbot/core/core_commands.py

```diff
@@ -3236,9 +3236,9 @@ async def autoimmune_checkimmune(
         """
 
         if await ctx.bot.is_automod_immune(user_or_role):
-            await ctx.send(_("They are immune"))
+            await ctx.send(_("They are immune."))
         else:
-            await ctx.send(_("They are not Immune"))
+            await ctx.send(_("They are not Immune."))
```

Comment:
This should probably be lower-case i and not upper-case.

Reply:
fixed. 👍